src/lib/getVersion.ts

```diff
@@ -0,0 +1,16 @@
+import * as Moment from 'moment';
+
+/**
+ * Get version from config.
+ *
+ * @param {String} version configuration object
+ *
+ * @returns {String} version string
+ */
+export default function getVersion(version: string): string {
+  if (version === 'latest') {
+    return Moment().unix().toString();
```

Comment:
if this is the only place you're using moment I'd suggest ditching for their one liner - https://github.com/moment/moment/blob/497f918515ae6ab900f008f19523b1e4ae5e2450/src/lib/moment/to-type.js#L6